pkg/client/admin.go

```diff
@@ -27,13 +27,21 @@ type ChildWorkflowOpts struct {
 	AdditionalMetadata *map[string]string
 }
 
+type Workflow struct {
```

Comment:
```suggestion
type WorkflowRun struct {
```

To keep naming consistent, instances of workflows should be `WorkflowRun` or we make it clear that it's a run when triggering via `RunWorkflow`.